fe/.eslintrc.js

```diff
@@ -26,10 +33,38 @@ module.exports = {
     // Ignore `import` statements with Webpack `@` alias in path
     'import/no-unresolved': ['error', { ignore: ['^@'] }],
 
+    // Set maximum line length to 120 columns if line doesn't contain is not url, string, template literal or regex
+    'max-len': ['error', {
+      'code': 120,
+      'ignoreUrls': true,
+      'ignoreStrings': true,
+      'ignoreTemplateLiterals': true,
+      'ignoreRegExpLiterals': true
+    }],
+
     // Do not permit `console` references or `debugger` statements in production bundle
     'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
 
+    // Quotes should be single if there is no template literals or quote to be escaped
+    'quotes': [
+      'error',
+      'single',
+      { 'avoidEscape': true, allowTemplateLiterals: true }
```

Comment:
`allowTemplateLiterals` – why do we want this option enabled?

Reply:
I agree that we do not need it.

Reply:
Aren't we planning to use template literals? 
![image](https://user-images.githubusercontent.com/16149737/58259579-80650980-7d7d-11e9-8250-c36e67443cc4.png)